app/decorators/usps_decorator.rb

```diff
@@ -16,6 +16,6 @@ def button
   private
 
   def letter_already_sent?
-    @idv_session.address_verification_mechanism == 'usps'
```

Comment:
In a follow-up PR, we should replace all instances of this check with the new one we created here. I just remembered that we're not actually sending the letter anywhere. Before you added the ability to limit the amount of letters a user can send, this session key was the only way to determine if the USPS method had been chosen. However, now that we keep track of when letters are sent in the DB, we can use that instead. Does that make sense?